packages/odyssey-react-mui/src/Badge.tsx

```diff
@@ -49,20 +49,21 @@ const Badge = ({
 }: BadgeProps) => {
   const odysseyDesignTokens = useOdysseyDesignTokens();
 
-  const greaterThanZeroContentMax = badgeContentMax > 0 ? badgeContentMax : 1;
-  const threeDigitLimitedMax =
-    greaterThanZeroContentMax > 999 ? 999 : greaterThanZeroContentMax;
-  const isOverContentMax = Boolean(
-    badgeContent && badgeContent > threeDigitLimitedMax
-  );
-  const overContentMaxMessage = `${greaterThanZeroContentMax}+`;
-  const formattedContent = isOverContentMax
-    ? overContentMaxMessage
-    : badgeContent;
-  const contentIsLongerThanOneChar = formattedContent?.toString()?.length > 1;
+  const renderBadge = useMemo(() => {
+    const greaterThanZeroContentMax = badgeContentMax > 0 ? badgeContentMax : 1;
+    const threeDigitLimitedMax =
+      greaterThanZeroContentMax > 999 ? 999 : greaterThanZeroContentMax;
+    const isOverContentMax = Boolean(
+      badgeContent && badgeContent > threeDigitLimitedMax
+    );
+    const overContentMaxMessage = `${greaterThanZeroContentMax}+`;
+    const formattedContent = isOverContentMax
+      ? overContentMaxMessage
+      : badgeContent;
 
-  const badgeStyles = useMemo<CSSProperties>(
-    () => ({
+    const contentIsLongerThanOneChar = formattedContent?.toString()?.length > 1;
```

Comment:
Isn't `formattedContent` always a string? If it's undefined, you should still be able to do `?.length` because of the question mark.

But `null > 1` and `undefined > 1` don't work the way you expect in JS, so I'd recommend not doing the check if it's null or undefined in the first place.